### PR TITLE
Migration: PHP8 improvements

### PR DESCRIPTION
**Pull request by daniwe4**

### CODE REVIEW

File: Services/Migration/test/ilServicesMigrationSuite.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+class ilServicesMigrationSuite extends TestSuite
+{
+    public static function suite()
+    {
+        $suite = new ilServicesMigrationSuite();
+
+        return $suite;
+    }
+}
```
Comment (mjansenDatabay) [inline Services/Migration/test/ilServicesMigrationSuite.php]: `PSR-2 + X` ;-)

Comment (daniwe4) [reply Services/Migration/test/ilServicesMigrationSuite.php]: Did you mean the return type? In this case i fix it. ;)

Comment (mjansenDatabay) [reply Services/Migration/test/ilServicesMigrationSuite.php]: I meant the missing `\n` at the end of the file.

Comment (daniwe4) [reply Services/Migration/test/ilServicesMigrationSuite.php]: Ok, thank you. I added a new line, but github doesn't show this correctly.

Comment (mjansenDatabay) [reply Services/Migration/test/ilServicesMigrationSuite.php]: Looks good now ^^

File: Services/Migration/DBUpdate_3136/classes/class.ilDBUpdate3136.php [hunk 2]
```diff
@@ -1,5 +1,18 @@
-<?php
-/* Copyright (c) 1998-2010 ILIAS open source, Extended GPL, see docs/LICENSE */
+<?php declare(strict_types=1);
+
+/******************************************************************************
```
Comment (mjansenDatabay) [inline Services/Migration/DBUpdate_3136/classes/class.ilDBUpdate3136.php]: According to our `Rector` in 'CI/Rector/ChangeLicenseHeader.php' the license header has the wrong format:

```php
/**
 * This file is part of ILIAS, a powerful learning management system
 * published by ILIAS open source e-Learning e.V.
 *
 * ILIAS is licensed with the GPL-3.0,
 * see https://www.gnu.org/licenses/gpl-3.0.en.html
 * You should have received a copy of said license along with the
 * source code, too.
 *
 * If this is not the case or you just want to try ILIAS, you'll find
 * us at:
 * https://www.ilias.de
 * https://github.com/ILIAS-eLearning
 *
 *********************************************************************/
```

File: Services/Migration/test/ilServicesMigrationSuite.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+/******************************************************************************
```
Comment (mjansenDatabay) [inline Services/Migration/test/ilServicesMigrationSuite.php]: See my comment above, the licese header differs from the definition in our `Rector`.

Comment (daniwe4) [reply Services/Migration/test/ilServicesMigrationSuite.php]: Thank you for the hint. I didn't know where to pick it. Hopefully now it is correct.

File: Services/Migration/DBUpdate_3136/classes/class.ilDBUpdate3136.php (hunk 2 above)
Comment (daniwe4) [reply Services/Migration/DBUpdate_3136/classes/class.ilDBUpdate3136.php]: Thank you for the hint. I didn't know where to pick it. Hopefully now it is correct.

**Pull request merged**